pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -190,3 +214,19 @@ func scoreToSeverity(score float64) types.Severity {
 		return types.SeverityUnknown
 	}
 }
+
+func saveAdvisories(dbc db.Operation, tx *bolt.Tx, vulnID string, details map[string]types.VulnerabilityDetail) error {
+	for _, source := range sources {
+		d, ok := details[source]
+		if !ok {
+			continue
+		}
```

Comment:
We should just iterate `details`.
```suggestion
	for _, d := range details {
```